Add: God Forsaken Abomination (Wood Sets)

wawwior opened the pull request:
Some experimental stuff for registries i added in case you want it.
It makes registering new wood sets very easy, but accessing the blocks looks a bit tedious.

wawwior committed:
Add: God Forsaken Abomination

Added WoodSet.java
Added WoodBlockType.java
Implemented both

ThatJadenXgamer commented:
this actually looks very interesting and really useful!
one thing I would suggest adding is some kind of bambooBuilder since smokestalk doesn't really have a hyphae-like block

wawwior committed:
Add: Smokestalk WoodSet

Maybe something like this?
SmokestalkBlock and SmokestalkPlantBlock could be integrated better if
more generic :3

wawwior commented:
Something like i committed or...

wawwior committed:
Fix: Better Naming

ThatJadenXgamer commented:
Yeah something like that works
